crates/wash/src/cli/cmd/dev/mod.rs

```diff
@@ -119,6 +119,10 @@ pub struct DevCommand {
     /// (useful for airgapped or disconnected environments)
     #[clap(long = "skip-fetch")]
     pub skip_wit_fetch: bool,
+
+    /// Optional mirror to use for downloading wadm, wasmCloud, and NATS binaries
+    #[clap(long = "github-mirror", env = "WASH_GITHUB_MIRROR")]
+    pub github_mirror: Option<String>,
```

Comment:
Should these flags parse to a `Url` type of some kind rather than a string or is that just too complicated?